skypy/galaxies/stellar_mass.py

```diff
@@ -74,3 +75,58 @@ def schechter_smf_mass(redshift, alpha, m_star, m_min, m_max, size=None,
     m = schechter(alpha, x_min, x_max, resolution, size=size, scale=m_star)
 
     return m
+
+
+def schechter_smf_parameters(active_parameters, fsatellite, fenvironment):
```

Comment:
I know a bit picky but maybe change the order of `fsatellite` and `fenvironment` to match the order in equation (15) of your paper. You do not need to change this but just an idea to be consistent.